opcua/common/subscription.py

```diff
@@ -201,22 +201,45 @@ def _get_node(self, nodeid):
     def _get_filter_from_event_type(self, eventtype):
         eventtype = self._get_node(eventtype)
         evfilter = ua.EventFilter()
-        for property in get_event_properties_from_type_node(eventtype):
+        evfilter.SelectClauses = self._select_clauses_from_evtype(eventtype)
+        evfilter.WhereClause = self._where_clause_from_evtype(eventtype)
+        return evfilter
+
+    def _select_clauses_from_evtype(self, evtype):
+        clauses = []
+        for property in get_event_properties_from_type_node(evtype):
             op = ua.SimpleAttributeOperand()
-            op.TypeDefinitionId = eventtype.nodeid
+            op.TypeDefinitionId = evtype.nodeid
             op.AttributeId = ua.AttributeIds.Value
             op.BrowsePath = [property.get_browse_name()]
-            evfilter.SelectClauses.append(op)
-        return evfilter
+            clauses.append(op)
+        return clauses
+
+    def _where_clause_from_evtype(self, evtype):
```

Comment:
I think this needs to take a list of event types so that it's possible to subscribe to multiple event types with a single filter and the WhereClause.